src/bisulfite_scoring.hpp

```diff
@@ -0,0 +1,133 @@
+// ==========================================================================
+//                                  lambda
+// ==========================================================================
+// Copyright (c) 2019, Sara Hetzel <hetzel @ molgen.mpg.de>
+// Copyright (c) 2016-2019, Knut Reinert and Freie Universität Berlin
+// All rights reserved.
+//
+// This file is part of Lambda.
+//
+// Lambda is Free Software: you can redistribute it and/or modify it
+// under the terms found in the LICENSE[.md|.rst] file distributed
+// together with this file.
+//
+// Lambda is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//
+// ==========================================================================
+// Scoring schemes for bisulfite converted data on dna5 alphabet
+// ==========================================================================
+
+#pragma once
+
+#include <range/v3/algorithm/copy.hpp>
+
+#include <seqan3/alignment/scoring/scoring_scheme_base.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/range/shortcuts.hpp>
+#include <seqan3/std/algorithm>
+
+namespace seqan3
+{
```

Comment:
This should just be in Lambda's namespace. [SeqAn3 actually states that you should never open namespace seqan3 in your app :wink: ]

Reply:
Well, I must have missed this :stuck_out_tongue_winking_eye: